### PR TITLE
feat: refactor project structure and dependencies

### PR DESCRIPTION
**Pull request by anqorithm**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
         <version>21.0.2</version>
     </parent>
 
-    <groupId>com.cevher.keycloak</groupId>
+    <groupId>com.anqorithm.keycloak</groupId>
```
Comment (cevheri) [inline pom.xml:13]: this is your private name

File: src/main/java/com/anqorithm/keycloak/CustomEventListenerProviderFactory.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package com.cevher.keycloak;
+package com.anqorithm.keycloak;
```
Comment (cevheri) [inline src/main/java/com/anqorithm/keycloak/CustomEventListenerProviderFactory.java:1]: this is your private name

**Review by cevheri — COMMENTED** [review]
Hi @anqorithm 
can you change your private names(package name)

Thanks for your contribution

File: pom.xml (hunk 1 above)
Comment (anqorithm) [reply pom.xml:13]: Sure, I forgot it.

File: src/main/java/com/anqorithm/keycloak/CustomEventListenerProviderFactory.java (hunk 2 above)
Comment (anqorithm) [reply src/main/java/com/anqorithm/keycloak/CustomEventListenerProviderFactory.java]: Done

**Comment by cevheri** [comment]
Hi @anqorithm 
I made some a little bit improvements

Thanks for your contributing

**Comment by anqorithm** [comment]
> Hi @anqorithm I made some a little bit improvements
> 
> Thanks for your contributing

I really appreciate it, 
This package is really awsome, 
thanks 💙

**Pull request merged**